pcf_test.go

```diff
@@ -109,28 +111,30 @@ var Data = []byte(`{
 		}
 		]}`)
 
-/*func TestUpdateConfig(t *testing.T) {
-	var nrp protos.NetworkSliceResponse
-	err := json.Unmarshal(Data, &nrp)
-	if err != nil {
-		panic(err)
-	}
-	var Rsp chan *protos.NetworkSliceResponse
-	Rsp = make(chan *protos.NetworkSliceResponse)
-	go func() {
-		Rsp <- &nrp
-	}()
-	go func() {
-		AMF.UpdateConfig(Rsp)
-	}()
-	time.Sleep(2 * time.Second)
-	if factory.AmfConfig.Configuration.SupportTAIList != nil &&
-		len(factory.AmfConfig.Configuration.SupportTAIList) == 2 {
-		fmt.Printf("test passed")
-	} else {
-		t.Errorf("test failed")
+/*
+	func TestUpdateConfig(t *testing.T) {
+		var nrp protos.NetworkSliceResponse
+		err := json.Unmarshal(Data, &nrp)
+		if err != nil {
+			panic(err)
+		}
+		var Rsp chan *protos.NetworkSliceResponse
+		Rsp = make(chan *protos.NetworkSliceResponse)
+		go func() {
+			Rsp <- &nrp
+		}()
+		go func() {
+			AMF.UpdateConfig(Rsp)
+		}()
+		time.Sleep(2 * time.Second)
+		if factory.AmfConfig.Configuration.SupportTAIList != nil &&
+			len(factory.AmfConfig.Configuration.SupportTAIList) == 2 {
+			fmt.Printf("test passed")
+		} else {
+			t.Errorf("test failed")
+		}
 	}
-}*/
+*/
```

Comment:
Why not deleting this function instead of commenting it out?

Reply:
Please see my comment above